manifests/argo-cd/resource.customizations.health.networking.k8s.io_Ingress

```diff
@@ -0,0 +1,17 @@
+hs = {}
+if obj.metadata.annotations["nginx.org/mergeable-ingress-type"] == "minion" then
+  hs.status = "Healthy"
+  return hs
+end
+if obj.metadata.annotations["nginx.org/mergeable-ingress-type"] == "master" then
+  hs.status = "Healthy"
+  return hs
+end
```

Comment:
is there a way in lua to just check if `obj.metadata.annotations["nginx.org/mergeable-ingress-type"]` exists?
anyway - really impressive lua stuff. well done.

Reply:
i can try to figure out it there is, do you mean so that we can just check if it exists and that will be the condition?

Reply:
yeah, instead of checking if its "minion", and later check if it's "master", and basically do the same in both conditions.